softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/factories/ResponseFactory.java

```diff
@@ -0,0 +1,26 @@
+package com.softgroup.common.protocol.factories;
+
+import com.softgroup.common.protocol.*;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 29.03.17.
+ */
+
+public class ResponseFactory {
+
+    public static <T extends ResponseData> Response<T> createResponse(ActionHeader header, T data){
```

Comment:
@AlexKazmirchuk First parameter must be Request. We can not create response from header. Only from request